tests/ProtocolShareReserve.ts

```diff
@@ -0,0 +1,101 @@
+import { FakeContract, smock } from "@defi-wonderland/smock";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { ethers, upgrades } from "hardhat";
+
+import { convertToUnit } from "../helpers/utils";
+import { ComptrollerInterface, IRiskFund, MockToken, PoolRegistryInterface, ProtocolShareReserve } from "../typechain";
+
+let mockDAI: MockToken;
```

Comment:
nit: I like to scope these variables as close as possible so within the closest describe or test block.

You can return these values from the fixture to keep them scoped closely